lab-ronoh/lib/task-crud.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+// const debug = require('debug')('ronito: task-crud');
+const mongoose = require('mongoose');
+mongoose.Promise= require('bluebird');
```

Comment:
There are lots of conflicting ways to do this. Bluebird recommends using .promisifyAll, and I think that it's a little more up to date than the mongoose docs.